vumi_http_retry/tests/test_redis.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from twisted.trial.unittest import TestCase
+from twisted.internet.defer import inlineCallbacks
+
+from vumi_http_retry.tests.redis import create_client, zitems
+
+
+class TestRedis(TestCase):
+    @inlineCallbacks
+    def setUp(self):
+        self.redis = yield create_client()
+
+    @inlineCallbacks
+    def tearDown(self):
+        yield self.redis.delete('foo')
```

Comment:
> We might want to clean up everything under our test prefix in case something creates unexpected keys?
Not following, I don't think we have any prefix. I thought we were literally storing the key `foo`. Unless txredis is adding a prefix under the hood?

> I guess we're intentionally avoiding Vumi and FakeRedis? (I'm not against trying that approach here).
Err, I was under the impression that was the approach we were taking, but now that I think about it, we didn't explicitly decide on that. I'm happy either way, though avoiding the dependency does seem nicer.